packages/terra-modal/tests/nightwatch/modal-spec.js

```diff
@@ -1,144 +1,135 @@
-      .assert.elementPresent('div[role="document"]');
-  },
-
-  'role is overridable': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/override-role`)
-      .assert.elementPresent('div[role="button"]');
-  },
-
-  'append css class to modal': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/append-class`)
-      .expect.element('div[role="document"]').to.have.attribute('class').which.contains('modal-custom-class');
-  },
-
-  'append css class to overlay': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/append-class`)
-      .expect.element('[class*="ModalOverlay"]').to.have.attribute('class').which.contains('overlay-custom-class');
-  },
-
-  'content overflow example': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`)
-      .assert.elementPresent('#modal-overflow');
-  },
-
-  'modal is focusable': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/no-focusable-content`)
-      .expect.element('div[role="document"]').to.have.attribute('tabindex').which.equals('0');
-  },
-
-  'background does not scroll when modal is open': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`)
-      .expect.element('html').to.have.attribute('style').which.equals('overflow: hidden;');
-
-    browser
-      .click('[class*="ModalOverlay"]')
-      .expect.element('html').to.have.attribute('style').which.equals('');
-  },
-
-  'isScrollable default value': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/content-overflow`)
-      .assert.elementPresent('#modal-overflow');
-  },
-
-  'isScrollable={false}': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/scrollable-false`)
-      .assert.elementPresent('#modal-scroll-false');
-  },
-
-  'isScrollable={true}': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/scrollable-true`)
-      .assert.elementPresent('#modal-scroll-true');
-  },
-
-  'custom props': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/custom-props`)
-      .assert.attributeEquals('div[role="document"]', 'id', 'custom-props');
-  },
-};
+// // eslint-disable-next-line import/no-extraneous-dependencies
+// const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+//
+// module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
```

Comment:
What's going on with this test? Are we gonna hold off on it due to stability of the test?

Reply:
Should be fixed in this [commit](https://github.com/cerner/terra-core/pull/751/commits/3e3f95b0c3ddb29f70eb7128aafde48d148695bb#diff-0257b38edc76b2ad44acf8f8871dc0f7R5)